ui/src/app/shared/revision-help-icon.tsx

```diff
@@ -0,0 +1,5 @@
+import {HelpIcon} from 'argo-ui';
```

Comment:
All shared components are in `ui/src/app/shared/components`. Please move `RevisionHelpIcon` there too.